README.md

```diff
@@ -47,6 +47,12 @@ _Long-form documentation for this section can be found
 [here](docs/build_and_load.md). Additional information about **QC 
 metrics** can be found [here](docs/qc_metrics.md)._
 
+> [!NOTE]
+> As of version 2.4.X, the phg utilizes a new version of anchorwave(1.2.3).
+> This changes how ASM coordinates are handled. 
+> If you are using old MAF files generated either from anchorwave 1.2.2 or from PHGv2 version 2.3 or eariler, 
+> please use the --legacy-maf-file flag for create-maf-vcf.  More information can be found [here](docs/build_and_load.md).
+
```

Comment:
I like the new wording.  

But I'm still concerned users may not grasp that running with the new phg version does not mean that any new maf files created from align-assemblies will be with the newer version of anchorwave.  We need to make it clear to them somewhere in the documentation that they must update their conda environment with the new version of anchorwave as is specified in the phg_environment.yml file.  Just re-running setup-environment won't fix this either as it skips the setup if the "phgv2-conda" env already exists.   They will need to do a "conda update ..."

@btmonier Do you have thoughts on where best this information should be documented?

Reply:
I can add in a note that it is recommended to remake your conda environment, but users should really be doing this every time they are downloading a new version anyways to fully rule out inconsistencies.  I am fully expecting we get some user requests asking why the coordinates are different, but unfortunately there really is not a good way to force people to do it right.  I am also going to put out an announcement on the discussions page once this branch is merged in.

Reply:
@lynnjo @zrm22 - if anything, we could also add a note in the [Setup Environment ](https://phg.maizegenetics.net/build_and_load/#set-up-conda-environment) section of the docs.

Reply:
@btmonier I think adding a note in the SetupEnvironment section is a good idea.